test/e2e/specs/wp-media-editor-spec.js

```diff
@@ -35,14 +33,6 @@ describe( `[${ host }] Media: Edit Media (${ screenSize }) @parallel @jetpack`,
 			await loginFlow.loginAndSelectMySite();
 		} );
 
-		step( "Can see a 'Media' option", async function () {
-			const sideBarComponent = await SideBarComponent.Expect( driver );
-			return assert(
-				await sideBarComponent.mediaOptionExists(),
-				'The settings menu option does not exist'
-			);
-		} );
-
```

Comment:
I might have missed it elsewhere in this patch - we don't need this test anymore because the Media option is a top level item in the sidebar and so its presence is checked elsewhere, is this correct?

Reply:
Yes, that is correct. I meant to clarify that